### ISSUE TITLE
Custom properties inside CSS functions

### ISSUE THREAD

**Issue by sandrojohanides** [braver/SublimeSass#48]
See various colors of `--properties`.

Might be related: https://github.com/braver/SublimeSass/issues/40

![screen shot 2018-11-27 at 13 34 31](https://user-images.githubusercontent.com/8327913/49082141-47722f00-f249-11e8-90d3-41505206641a.png)
```scss
content: var(--progress);
content: url(var(--input));
content: url(var(--progress));
content: calc(var(--progress));
content: translateX(calc(var(--x)));
content: translateX(calc(var(--text)));
content: translateX(calc(var(--input)));
content: translateX(calc(var(--progress)));
```

**Comment by braver** [comment]
It's indeed related to #40, and I think it will be solved by simplifying the function arguments. Let's keep this open here as a useful check.

**Comment by braver** [comment]
This one should be fixed, #40 has some cases that need work.
